lib/safira_web/controllers/redeem_controller.ex

```diff
@@ -25,23 +28,26 @@ defmodule SafiraWeb.RedeemController do
 
   defp company_aux(conn, redeem_params, user) do
     redeem_params = Map.put(redeem_params, "badge_id", user.company.badge_id)
+
     with {:ok, %Redeem{} = _redeem} <- Contest.create_redeem(redeem_params) do
       conn
       |> put_status(:created)
-      |> json(%{redeem: "Badge redeem successfully"})
+      |> json(%{redeem: "Badge redeem successfully. Tokens added to your balance"})
     end
```

Comment:
Since we're not returning the new balance, will the frontend need to do a new call to retrieve all of the user's info? @joaop21

Reply:
yes, the frontend need to do a new call to retrieve all of the attendee's info. But this message is fine because this POST added a redeem and updated the attendee's balance.